lib/clients/github.go

```diff
@@ -167,7 +171,12 @@ func (g realGHClient) request(f func() (interface{}, *github.Response, error)) (
 	b := backoff.NewExponentialBackOff()
 	b.MaxElapsedTime = g.config.GetTimeout()
 
-	er := backoff.Retry(op, b)
+	er := backoff.RetryNotify(op, b, func(err error, duration time.Duration) {
+		duration /= 1000000 // Convert nanoseconds to milliseconds
+		duration *= 1000000 // Convert back so it appears correct
```

Comment:
Why are you doing and undoing this? You end up with the exact same value. And actually, if you have fewer than 1000000 nanoseconds, eg 5 nano seconds, this will clip and give you 0s. Finally, you don't know the units of this duration, it could be nanoseconds, or it could be days.

https://play.golang.org/p/yqQ3VNwUJQ